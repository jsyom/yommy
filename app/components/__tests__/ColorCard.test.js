import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ColorCard from '../ColorCard';

const props = { color: '#2C3B40', label: 'DarkSlateGray' };

test('ColorCard component should render as expected', () => {
	const component = shallow(<ColorCard {...props} />);
	const tree = toJson(component);
	expect(component.contains('#2C3B40')).toBe(true);
	expect(tree).toMatchSnapshot();
});

test('ColorCard component should render as expected', () => {
	const component = shallow(<ColorCard {...props} />);
	const tree = toJson(component);
	expect(component.contains('#2C3B40')).toBe(true);
	expect(tree).toMatchSnapshot();
});
