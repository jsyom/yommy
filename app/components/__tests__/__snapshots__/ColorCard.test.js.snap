// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorCard component should render as expected 1`] = `
<div
  className="single-card"
  style={
    Object {
      "backgroundColor": "#2C3B40",
    }
  }
>
  <div
    className="label"
  >
    <p>
      <b>
        HEX: 
      </b>
      #2C3B40
    </p>
    <p>
      <b>
        Color: 
      </b>
      DarkSlateGray
    </p>
  </div>
</div>
`;
